### PR TITLE
Refereshed links

### PR DESCRIPTION
**Pull request by EgonMarmol**
Updated a handful of old links on the Securing page.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_docs/configuration/securing.markdown [hunk 1]
```diff
@@ -16,8 +16,8 @@ Here's the summary of what you *must* do to secure your Home Assistant system:
 
 If you want secure remote access, the easiest option is to use [Home Assistant cloud](/cloud/) by which you also [support](https://www.nabucasa.com/about) the founders of Home Assistant. 
 
-Other options are to use TLS/SSL via the add-ons [Duck DNS integrating Let's Encrypt](/integrations/duckdns/) or [Let’s Encrypt](/docs/ecosystem/certificates/lets_encrypt/). 
-To expose your instance to the internet, use a [VPN](https://pivpn.dev/), or an [SSH tunnel](/blog/2017/11/02/secure-shell-tunnel/). Make sure to expose the used port in your router. 
+Other options are to use TLS/SSL via the add-ons [Duck DNS integrating Let's Encrypt](/integrations/duckdns/) or [Let’s Encrypt](https://community.home-assistant.io/t/installing-tls-ssl-using-lets-encrypt/196975).
```
Comment (frenck) [inline source/_docs/configuration/securing.markdown]: That is actually a community guide and should be removed.

**Review by frenck — CHANGES_REQUESTED** [review]
See comment above

File: source/_docs/configuration/securing.markdown (hunk 1 above)
Comment (EgonMarmol) [reply source/_docs/configuration/securing.markdown]: @frenck Thanks for the heads up. The original link redirected to the community URL. Do you want me to remove the reference to Let's Encrypt entirely, or change it to something else like this blog post: https://www.home-assistant.io/blog/2017/09/27/effortless-encryption-with-lets-encrypt-and-duckdns/

Comment (frenck) [reply source/_docs/configuration/securing.markdown]: That blog post is heavily outdated as well. Let's remove it.

Comment (EgonMarmol) [reply source/_docs/configuration/securing.markdown]: @frenck Thanks for clarifying. I will fix.

Comment (EgonMarmol) [reply source/_docs/configuration/securing.markdown]: @frenck The changes have now been applied.

**Review by frenck — APPROVED** [review]
Thanks, @EgonMarmol 👍

**Pull request merged**